src/main/java/io/epirus/console/deploy/DeployCommand.java

```diff
@@ -211,6 +211,12 @@ private void runGradle(Path runLocation) throws Exception {
     private void executeProcess(File workingDir, String[] command) throws Exception {
         ProcessBuilder processBuilder = new ProcessBuilder(command);
         processBuilder.environment().put("DEPLOY_NETWORK", network.getNetworkName());
+        processBuilder
```

Comment:
this should not have been implemented like this methinks. The actual command needs to decide whether to use an env var or the default config - not mess with the environment.

hack of the year right here